src/components/form/PDFFunctions.js

```diff
@@ -125,14 +125,25 @@ export function getTextHeight(text,  fontSize = font.font_size.DEFAULT_FONT_SIZE
  * @param  {jsPDF}  doc           PDF document object
  * @param  {String} text          Text to be split into multiple lines
  * @param  {Number} maxLineWidth  The maximum width of the text
+ * @param  {Number} maxLines      The maximum amount of lines paragraph can hold
  * @param  {String} font          = font.font_type.DEFAULT, text font
  * @return {Array}                Array of string lines split by maxLineWidth
  */
- export function createParagraph(doc, text, maxLineWidth, fontType = font.font_type.DEFAULT, fontSize = font.font_size.MINIMUM_FONT_SIZE) {
-  //var textLines = doc.splitTextToSize(text, maxLineWidth);
-  //console.log(fontType);
-  return  doc.setFontSize(fontSize)
-             .splitTextToSize(text, maxLineWidth);
+ export function createParagraph(doc, text, maxLineWidth, maxLines,fontType = font.font_type.DEFAULT, fontSize = font.font_size.MINIMUM_FONT_SIZE) {
```

Comment:
The only thing I would mention here is we may want to set maxLines to a default value of 0. If not, we will have to pass 0 every time we call createParagraph, which may cause errors everywhere else we call createParagraph, since that parameter maxLines is now missing. If you haven't already, make sure to test this!

Reply:
Good idea, I am just a little confused on we can set 

`createParagraph(doc, text, maxLineWidth, maxLines = page.descriptions.SIZE, fontType = font.font_type.DEFAULT, fontSize = font.font_size.MINIMUM_FONT_SIZE) `

Would we then set **SIZE** to zero? But I dont think that'll work